frontend/src/store.ts

```diff
@@ -11,7 +11,7 @@ export const store = configureStore({
     stepper: stepperReducer,
     config: configReducer,
     jiraVerifyResponse: jiraVerifyResponseReducer,
-    pipelineToolVerifyResponse: pipelineToolResponseReducer,
+    pipelineToolVerifyResponse: pipelineToolVerifyResponseReducer,
     sourceControlVerifyResponse: sourceControlVerifyResponseReducer,
     saveMetricsSetting: saveMetricsSettingReducer,
   },
```

Comment:
and I think board, pipeline tool, source control are similar,could we build a new card to refactor it ?

Reply:
I think so about that!